### PR TITLE
Update documentation for PodSecurityPolicy removal

### PR DESCRIPTION
**Pull request by sftim**
Really mark PSP as removed.

- cut out some references to PSP
- treat PSA as stable (but hint that older releases might provide it - which they do, in beta)

### Pages changed

1. [original](https://k8s.io/docs/concepts/security/pod-security-admission/) / [preview](https://deploy-preview-37733--kubernetes-io-vnext-staging.netlify.app/docs/concepts/security/pod-security-admission/)
1. [original](https://k8s.io/docs/tasks/configure-pod-container/enforce-standards-admission-controller/) / [preview](https://deploy-preview-37733--kubernetes-io-vnext-staging.netlify.app/docs/tasks/configure-pod-container/enforce-standards-admission-controller/)
1. [original](https://k8s.io/docs/reference/access-authn-authz/admission-controllers/) / [preview](https://deploy-preview-37733--kubernetes-io-vnext-staging.netlify.app/docs/reference/access-authn-authz/admission-controllers/)
1. [original](https://k8s.io/docs/tasks/configure-pod-container/enforce-standards-namespace-labels/) / [preview](https://deploy-preview-37733--kubernetes-io-vnext-staging.netlify.app/docs/tasks/configure-pod-container/enforce-standards-namespace-labels/)
1. ~[original](https://k8s.io/docs/concepts/security/rbac-good-practices/) / [preview](https://deploy-preview-37733--kubernetes-io-vnext-staging.netlify.app/docs/concepts/security/rbac-good-practices/)~

/kind cleanup

### CODE REVIEW

File: content/en/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md [hunk 1]
```diff
@@ -4,19 +4,17 @@ reviewers:
 - tallclair
 - liggitt
 content_type: task
-min-kubernetes-server-version: v1.22
+min-kubernetes-server-version: v1.25
```
Comment (tengqm) [inline content/en/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md:7]: Why this change?

File: content/en/docs/tasks/configure-pod-container/enforce-standards-namespace-labels.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ reviewers:
 - tallclair
 - liggitt
 content_type: task
-min-kubernetes-server-version: v1.22
+min-kubernetes-server-version: v1.25
```
Comment (tengqm) [inline content/en/docs/tasks/configure-pod-container/enforce-standards-namespace-labels.md:7]: Same question here

File: content/en/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md:7]: Pod Security Admission is GA in Kubernetes 1.25

File: content/en/docs/tasks/configure-pod-container/enforce-standards-namespace-labels.md (hunk 2 above)
Comment (sftim) [reply content/en/docs/tasks/configure-pod-container/enforce-standards-namespace-labels.md:7]: Pod Security Admission is GA in Kubernetes 1.25

File: content/en/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md (hunk 1 above)
Comment (liggitt) [reply content/en/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md:7]: This page already indicates 1.25 is required for the v1 config, and links users to instructions that let them try/use this feature in alpha and beta state in 1.22, 1.23 and 1.24. What does this bit of the configuration do, and will it be misleading to label the whole page as being applicable only to 1.25+?

Comment (sftim) [reply content/en/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md:7]: I tweaked the page elsewhere.

Have a look at https://deploy-preview-37733--kubernetes-io-vnext-staging.netlify.app/docs/tasks/configure-pod-container/enforce-standards-admission-controller/#before-you-begin and https://deploy-preview-37733--kubernetes-io-vnext-staging.netlify.app/docs/tasks/configure-pod-container/enforce-standards-namespace-labels/#before-you-begin to see how these look now.


I think we should set `min-kubernetes-server-version` to the GA version. I can see why some people would prefer to set it to the version where the feature went beta / became enabled by default. Personally, I prefer to recommend the GA feature and then have a narrative explanation about what to do if your cluster is older.

Comment (sftim) [reply content/en/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md:7]: The `min-kubernetes-server-version` influences how a `{{% version-check %}}` shortcode renders.

File: content/en/docs/tasks/configure-pod-container/enforce-standards-namespace-labels.md (hunk 2 above)
Comment (sftim) [reply content/en/docs/tasks/configure-pod-container/enforce-standards-namespace-labels.md]: I've revised this; a preview should already be available.

File: content/en/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md]: I've revised again, a preview should already be available.

**Comment by sftim** [comment]
Hi @tallclair - thanks for the review / nudge

Could you check this again, sorry?

**Comment by reylejano** [comment]
The changes proposed are accurate
Thank you
/approve
